index.html

```diff
@@ -9298,6 +9298,279 @@
         </section>
       </div>
     </section>
+    <section id="rtcquicstream-interface-definition*">
+      <h3>Interface Definition</h3>
+      <div>
+        <pre class="idl">
+        [ Exposed=Window ]
+interface RTCQuicStream {
+    readonly        attribute RTCQuicTransport    transport;
+    readonly        attribute RTCQuicStreamState  state;
+    readonly        attribute unsigned long       bufferedAmount;
+    void                  close ();
```

Comment:
We need a way to send the fin bit.  Options:

Option A: finish() and close()
Option B: finishAndClose and closeWithoutFinishing(), or close(bool finish_too == true)

With unidirectional streams, B might work fine.   If we want to finish but stay open for reading with bidirectional, A might make more sense.

For now, I would prefer A and think more about B if/when we have unidirectional.

Reply:
The text now indicates that close() causes the F bit to be set on a STREAM frame.  Is this sufficient?

Reply:
I think the names should match what QUIC does:  finish() for the fit bit and perhaps reset() or close() for abrupt close (whether QUIC decides on reset or close or whatever).